File: test/fixtures/langchain-repo/lc_anthropic_chain.py
from langchain_anthropic import ChatAnthropic
from langchain_core.prompts import ChatPromptTemplate

model = ChatAnthropic(model="claude-sonnet-5")
prompt = ChatPromptTemplate.from_messages([("system", "Be terse.")])
chain = prompt | model

result = chain.invoke({"input": "hello"})

File: test/fixtures/langchain-repo/lc_anthropic_chain.ts
import { ChatAnthropic } from "@langchain/anthropic";
import { ChatPromptTemplate } from "@langchain/core/prompts";

const model = new ChatAnthropic({ model: "claude-sonnet-5" });
const prompt = ChatPromptTemplate.fromMessages([["system", "Be terse."]]);
const chain = prompt.pipe(model);
await chain.invoke({ input: "hello" });
